builtin/game/register.lua

```diff
@@ -79,8 +79,22 @@ local function check_modname_prefix(name)
 	end
 end
 
+local function check_node_list(list, field)
+	local t = type(list)
+	if t == "table" then
+		for _, entry in pairs(list) do
+			assert(type(entry) == "string" or type(entry) == "number",
```

Comment:
The "number" variant is not documented, hence should not be allowed as a type here.

Reply:
We're talking about node names here, right? I would say it's insanity to register and/or explicitly use node names of type "number".